src/source-af-packet.c

```diff
@@ -1388,14 +1548,168 @@ frame size: TPACKET_ALIGN(snaplen + TPACKET_ALIGN(TPACKET_ALIGN(tp_hdrlen) + siz
     return 1;
 }
 
+static int AFPComputeRingParamsV3(AFPThreadVars *ptv)
+{
+    ptv->req3.tp_block_size = getpagesize();
```

Comment:
Good catch, I lot a bigger value in a refactoring. Making it tunable is a good idea. It will be in the next PR.